src/main/java/org/isf/security/OHSimpleUrlAuthenticationSuccessHandler.java

```diff
@@ -27,6 +27,8 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
per project import ordering standards this import should appear after the `org` ones